templates/directory.php

```diff
@@ -536,8 +549,20 @@ function pmpromd_shortcode($atts, $content=null, $code="")
 									?>
 									<p class="pmpro_member_directory_<?php echo $field[1]; ?>">
 										<?php
-										if(is_array($meta_field) && !empty($meta_field['filename']) )
-										{
+										if( taxonomy_exists( $field[1] ) ){
```

Comment:
`$field[1]` is the name of the PMPro_Field object, not necessarily the name of the associated taxonomy. If we wanted to check if this field was associated with a taxonomy, I think we would need to get the PMPro_Field object (if it exists) and check the `$field->taxonomy` property.

Can test this by creating a field with a different `name` and `taxonomy` value and seeing that numbers still display on the Directory page.